docs/1.-How-to-Use-JPlag.md

```diff
@@ -151,12 +151,12 @@ try {
 
 ## Report File Generation
 
-After a JPlag run, a zipped result report is automatically created.
+After a JPlag run, a zipped result file is automatically created.
 The target location of the report can be specified with the `-r` flag.
 
 If the `-r` is not specified, the location defaults `result.jplag`. Specifying the `-r` flag with a path `/path/to/desiredFolder` results in the report being created as `/path/to/desiredFolder.jplag`.
 
-The report will always be zipped unless there is an error during the zipping process. If the zipping process fails, the report will be available in an unzipped version under the specified location.
+The report will always be zipped unless there is an error during the zipping process. If the zipping process fails, the report will be available in an unzipped version under the specified location. Note that the result file use the file ending `.jplag`.
```

Comment:
"files use" or "file uses"?
"file extension"? ;)

Reply:
Ah, a little bit too late. I will sneak it into your typo-related PR.